src/utils/viewer/viewer.cpp

```diff
@@ -105,6 +112,10 @@ class Viewer : public Magnum::Platform::Application {
   scene::ObjectControls controls_;
   Magnum::Vector3 previousPosition_;
 
+  std::pair<Magnum::Vector3, Magnum::Vector3> commandVelocity;
```

Comment:
Use a struct instead of the std::pair to increase the readability.  
(`a.linearVelocity` for example is clearer than`a.first`. :-) )